packages/loader/container-loader/src/utils.ts

```diff
@@ -27,9 +28,9 @@ export function parseUrl(url: string): IParsedUrl | undefined {
 
     const regex = /^\/([^/]*\/[^/]*)(\/?.*)$/;
     const match = regex.exec(parsed.pathname);
-
+    console.log("hi", match);
```

Comment:
can you please clean up commented code and console.log statements?

Reply:
yes i just created the draft cause it's been a while since i last worked on this